benchmarks/src/CMakeLists.txt

```diff
@@ -1,26 +1,45 @@
-cmake_minimum_required(VERSION 3.10) # Set the minimum required version of CMake
+cmake_minimum_required(VERSION 3.10)
 
-project(SerializationBenchmark) # Name your project
+project(SerializatonTwitterBenchmark) # Name your project
```

Comment:
I am not convinced that the `project` instructions are needed. My experience has been that you use one per project, and not one per directory (a given project can have several subdirectories).

This being said, I do not particularly care.